paasta_tools/chronos_serviceinit.py

```diff
@@ -250,13 +251,15 @@ def format_chronos_job_status(job, running_tasks, verbose=0):
         mesos_status_verbose = status_mesos_tasks_verbose(job["name"], get_short_task_id, tail_stdstreams)
         mesos_status = "%s\n%s" % (mesos_status, mesos_status_verbose)
     return (
-        "Config:     %(config_hash)s\n"
-        "  Status:   %(disabled_state)s\n"
+        "Job:     %(job_name)s\n"
+        "  Status:   %(disabled_state)s"
+        "  Launched by rerun: %(is_temporary)s\n"
```

Comment:
This is a bit verbose.

Instead can you make it say like:

    Job:     $name (Launched by paasta rerun)

And only have that in the parens if it needs to?

95% of the time this isn't a thing, so I don't think we need a line saying "no" all the time